src/components/legend/attributes.js

```diff
@@ -35,6 +35,13 @@ module.exports = {
     font: extendFlat({}, fontAttrs, {
         description: 'Sets the font used to text the legend items.'
     }),
+    orientation: {
+        valType: 'enumerated',
+        values: ['v', 'h'],
```

Comment:
`'v'` / `'h'` gets my vote - for parity with bar [`orientation`](https://github.com/plotly/plotly.js/blob/5601e69790d0fd1c9e9f4f5ded956f8c2230891f/src/traces/bar/attributes.js#L27).